libarchive/archive_write_set_format_iso9660.c

```diff
@@ -2553,7 +2553,7 @@ set_date_time(unsigned char *p, time_t t)
 static void
 set_date_time_null(unsigned char *p)
 {
-	memset(p, '0', 16);
+	memset(p, (int)'0', 16);
```

Comment:
This is silly and plainly wrong. The semantics of memset are specified to operate on characters and the use of int is only present for the historical reasons of implicit argument promotion.